### DISCUSSION TITLE
How can I reinitialize the lenis instance after destroy()?

### DISCUSSION THREAD

**Discussion by jovanovskajovana** [darkroomengineering/lenis#333] [Q&A] [answered]
I have a conflict with the iubenda library's 2nd Layer that is not scrolling due to the lenis scroll. The combination of stop() and start() doesn't help at all, only destroy() does, but then I can't restart the instance.

**Comment by thomaslanciaux** [comment]
Have you tried to add the `data-lenis-prevent` attribute to the container you want to exclude from lenis? 
This worked for me, hope it does for you

**Comment by michaelreinhard1** [comment]
> I have a conflict with the iubenda library's 2nd Layer that is not scrolling due to the lenis scroll. The combination of stop() and start() doesn't help at all, only destroy() does, but then I can't restart the instance.

I'm having the same issue. Did you find a solution?

**Comment by clementroche** [comment]
You can use `prevent` option:

Manually prevent scroll to be smoothed based on elements traversed by events. If true is returned, it will prevent the scroll to be smoothed. Example: ```(node) =>  node.classList.contains('cookie-modal')```

**Reply by michaelreinhard1** [reply]
Hey, I'm using Lenis in Nuxt 3 and set it up as explained in the docs. It works fine on the first page load but when navigating between pages the scroll lags and flickers near the end of the page. 

I suspect the problem is caused by multiple Lenis instances overlapping since I'm initializing it on every page. I tried using the `resize()` function, but it didn’t fix the issue.

My idea is to destroy Lenis when leaving a page and reinitialize it on the next one. How can I properly destroy and reinitialize Lenis?

**Reply by clementroche** [reply]
destroy lenis -> lenis.destroy()

reinitialize Lenis -> new Lenis()

**Comment by dangelion** [comment]
I'm trying something like this and it works:

```
  const lenis = new Lenis({
    prevent: (node) => {
      return node.closest('#iubenda-iframe');
    }
  });
```

**Comment by clementroche** [comment, marked as answer]
you can use `allowNestedScroll` option
